pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -680,6 +692,34 @@ def read_mwa_corr_fits(
                 (self.Nblts, self.Nfreqs, self.Npols)
             )
 
+            # divide out digital gains
+            if remove_dig_gains:
+                # get gains for included coarse channels
+                coarse_inds = np.where(np.isin(coarse_chans, included_coarse_chans))[0]
+                print("coarse_inds: " + str(coarse_inds))
```

Comment:
I think this print statement should be removed.

Reply:
Thanks :)